modules/k8s-cluster/main.tf

```diff
@@ -76,6 +82,38 @@ resource "aws_cloudformation_stack" "worker-nodes" {
   depends_on = ["aws_eks_cluster.eks-cluster"]
 }
 
+resource "aws_cloudformation_stack" "worker-nodes-per-az" {
+  count         = "${length(data.aws_subnet.private_subnets.*.id)}"
```

Comment:
I didn't think this worked...I didn't think terraform let you dynamically count based on size of lists from variables...?

Reply:
urgh I can never predict ahead of time when this works and when it doesn't.  but I can change it to the static variable if you prefer. it's frustrating not being able to just run a plan and check